src/Makefile.am

```diff
@@ -73,6 +74,7 @@ fwts_SOURCES = main.c 				\
 	acpi/erst/erst.c			\
 	acpi/facs/facs.c 			\
 	acpi/fadt/fadt.c 			\
+	sbbr/fadt/fadt.c 			\
```

Comment:
is it difficult to make sbbr specific changes in acpi/fadt/fadt.c?If there a lot of changes, that's a valid argument to make when it is sent to maintainers.